sklearn/datasets/_base.py

```diff
@@ -1100,6 +1100,9 @@ def load_linnerud(*, return_X_y=False, as_frame=False):
             .. versionadded:: 0.20
 
     (data, target) : tuple if ``return_X_y`` is True
+        Returns a tuple of two ndarray of shape (n_samples, n_features)
+        A 2D array with each row representing one sample and each column
+        representing the features and/or target of a given sample.
```

Comment:
```suggestion
        Returns a tuple of two ndarrays or dataframe of shape
        `(n_samples, n_features)`. Each row represents one sample and
        each column represents the features in `X` and a target in `y` of a
        given sample.
```

Reply:
@glemaitre do you need me to change the description?

Reply:
Yes above your message you can see my suggestion. you can accept it by clicking on `Commit suggestion`

Reply:
Maybe it is worth that you take a read at the following short documentation if you are not familiar with the suggestion process available in GitHub: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

We usually rely on it to make changes in PRs.